src/ClientHandle.cpp

```diff
@@ -2320,6 +2320,15 @@ void cClientHandle::SendHealth(void)
 
 
 
+void cClientHandle::SendHideTitle(void)
```

Comment:
Oh, no. Its better to remove the SendResetTitle() and call it if SendSubtitle() is called with an empty string.